src/material/datepicker/datepicker.md

```diff
@@ -161,9 +161,9 @@ that point.
 
 <!-- example(datepicker-filter) -->
 
-In this example the user can back past 2005, but all of the dates before then will be unselectable.
-They will not be able to go further back in the calendar than 2000. If they manually type in a date
-that is before the min, after the max, or filtered out, the input will have validation errors.
+Sometimes we need a bit more control about which dates are valid or not. In this example the user 
```

Comment:
I think the part about "Sometimes we need a bit more control about which dates are valid or not." is unnecessary here.

Reply:
okie.. let me remove it. got that part from [angular university](https://blog.angular-university.io/angular-material-datepicker/)